tests/README.md

```diff
@@ -2,27 +2,47 @@
 
 ## Overview
 
-This project contains UI smoke tests for Busola. Copy your cluster's kubeconfig file to [`fixtures/kubeconfig.yaml`](fixtures/kubeconfig.yaml).
+This project contains smoke and integration UI tests for Busola. Copy your cluster's kubeconfig file to [`fixtures/kubeconfig.yaml`](fixtures/kubeconfig.yaml).
 
 ## Installation
 
 To install dependencies, run the `npm install` command.
 
 ## Development
```

Comment:
What do you think about adding additional subtitles to make this more readable:

`### Run Cypress UI tests in the headless mode`
...
`### Run Cypress UI tests in the test runner mode`